web_src/js/features/common-global.js

```diff
@@ -201,13 +190,19 @@ export function initGlobalDropzone() {
           file.uuid = data.uuid;
           const input = $(`<input id="${data.uuid}" name="files" type="hidden">`).val(data.uuid);
           $dropzone.find('.files').append(input);
+          addUploadedFileToEditor(file.editor, file);
         });
         this.on('removedfile', (file) => {
           $(`#${file.uuid}`).remove();
+          if (!file.editor && (file.editor = getAttachedEasyMDE(this.element.parentElement.parentElement.querySelector('textarea')))) {
```

Comment:
Maybe `this.element.closest('parent').querySelector('textarea')` to make it not so tightly coupled to DOM structure.

Reply:
![image](https://user-images.githubusercontent.com/1255041/179430860-8a70555b-bc03-4ac0-8f79-1c53653f709b.png)

Reply:
`parent` of course should be replaced with a selector for a common parent node.

Reply:
Thanks, Fixed on [ad444d3](https://github.com/go-gitea/gitea/pull/20263/commits/ad444d305ddb67065e41474853eb6fb4a225b47d)